docs/.project/RELEASE_TEMPLATE.md

```diff
@@ -36,12 +36,12 @@ _To be done after code freeze to prepare and test the release._
 * [ ] close all issues which are solved by dependency updates
 * [ ] smoke test to verify all diagrams can be created
 * [ ] update [Release Info](https://github.com/camunda/camunda-modeler/blob/develop/client/src/plugins/version-info/ReleaseInfo.js)
-    * [ ] create a draft following [our guidelines](https://github.com/bpmn-io/internal-docs/tree/master/camunda-modeler#whats-new-communication) and based on priorities which were aligned with the team (PM, UX, and Engineering side)
+    * [ ] create a draft following [our guidelines](https://github.com/bpmn-io/internal-docs/blob/master/releases/modeler/CAMUNDA_MODELER.md#whats-new-communication) and based on priorities which were aligned with the team (PM, UX, and Engineering side)
    * [ ] create PR to merge the draft into `develop`. Assign to PM, UX and Engineering for review
 * [ ] update [`CHANGELOG`](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md)
 * [ ] verify `develop` is up to date with `master`: `git checkout master && git pull && git checkout develop && git merge master`
 * [ ] merge to master: `git checkout master && git merge develop`
-* [ ] create release candidate (`npm run release`), cf. [release schema](https://github.com/bpmn-io/internal-docs/tree/master/release-schema); e.g `v1.0.0-rc.0`
+* [ ] create release candidate (`lerna publish [prepatch|preminor|premajor] --preid rc`), cf. [release schema](https://github.com/bpmn-io/internal-docs/blob/master/releases/RELEASE_SCHEMA.md)
```

Comment:
We could think about adding this as a script to the package.json:
https://github.com/camunda/camunda-modeler/blob/develop/package.json#L28

We could: 
- add `release:rc` script
- or add parameters via command line and document it here: `npm run release -- preminor --preid rc`

Reply:
Yes, we would still need to pass it to the command actual version we want. I'm not sure if just passing `--preid` will apply it to the version selection menu. Maybe something we can try out